### ISSUE TITLE
error opening playlists from web

### ISSUE THREAD

**Issue by shape55** [gaqzi/chrome-open-in-spotify-client#1]
The extension works great w/ all open.spotify.com links I've tested but when it tries to load an embedded playlist from the web into the desktop player I get this error:

"There was a problem displaying this view!  This view either does not exist or an error has occurred."  If I disable the extension the playlists load on the web site as expected.

Here are a couple examples: 

https://www.reddit.com/r/spotify/comments/8opjis/back_to_the_80s_i_best_of_synthwave_and_retrowave/

https://www.reddit.com/r/spotify/comments/8ou5qh/mf_doom_48_timeless_songs_from_my_favorite_hiphop/

**Comment by gaqzi** [comment]
@shape55 thanks for the bug report, I've published an updated version that shouldn't touch embedded links. I've tested with both of your links and they've worked. 

Google said version 1.3 should be available within an hour or so. :)

**Comment by shape55** [comment]
Sure, no prob - thanks for the lightning fast fix!  I tested out those 2 links again using 1.3 and they load in the browser just fine now.  Great job with this extension.  I had tried a few other ones from the Chrome Web Store and this was the best one I found, even before you made this fix :)

In case you didn't see these, there are a handful of comments in the Support section of your extension: https://chrome.google.com/webstore/detail/open-in-spotify-desktop-c/okkdbmdhpgmajopdpmflkldkemcldnjd/support
